tests/ttnn/unit_tests/operations/eltwise/test_binary_bcast.py

```diff
@@ -17,16 +19,40 @@
         (torch.Size([5, 1, 1, 64]), torch.Size([1, 3, 128, 1])),
     ),
 )
-def test_binary_scalar_ops(input_shapes, device):
+@pytest.mark.parametrize(
```

Comment:
can you please also add a test where bcast must fail due to not following bcast rules?

Reply:
Sure!